FirebaseRemoteConfig/Sources/RCNConfigDBManager.m

```diff
@@ -898,7 +901,7 @@ - (void)loadPersonalizationWithCompletionHandler:(RCNDBLoadCompletion)handler {
     }
 
     if (handler) {
-      dispatch_async(dispatch_get_main_queue(), ^{
+      dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
```

Comment:
Note for reviewers. Validate if it affects calling a public callback. If yes, it's a breaking change and will require a major version bump.

Reply:
Just for additional info: 
1. There is same kind of callback in `- (void)loadMainWithBundleIdentifier` in this file.
In `- (void)loadMainWithBundleIdentifier` callback already looks like:
```
    if (handler) {
      dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
      fetchedConfig = fetchedConfig ? fetchedConfig : [[NSDictionary alloc] init];
      activeConfig = activeConfig ? activeConfig : [[NSDictionary alloc] init];
      defaultConfig = defaultConfig ? defaultConfig : [[NSDictionary alloc] init];
      handler(YES, fetchedConfig, activeConfig, defaultConfig);
    });
```
2. Another thought on this is that "Personalization" feature is in alpha now (afaik).

But if this change still needs major version bump, then I can move it to separate PR, to make main part sooner to merge.

Reply:
Thank you for pointing out to that the handler is already called on the global queue in some cases. Modifications in the file look good to me. 

cc @karenyz